tests/quick_tests/run.cmake

```diff
@@ -34,15 +34,6 @@ to the mailing list linked at http://www.dealii.org\n"
     )
 
   FOREACH(test ${ALL_TESTS})  
-    IF (${test} MATCHES "^affinity" AND NOT EXISTS ${test}-OK)
-      MESSAGE("
-The affinity test can fail when you are linking in a library like BLAS
-which uses OpenMP. Even without calling any BLAS functions, OpenMP messes
-with the thread affinity which causes TBB to run single-threaded only. You
-can fix this by exporting OMP_NUM_THREADS=1.\n"
```

Comment:
Ah, I think this gives a bit of context why we once thought that this test is a good idea. Let's keep it in mind for if/when we ever decide to resurrect the test.